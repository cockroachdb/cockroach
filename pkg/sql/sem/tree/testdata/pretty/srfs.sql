select * from
   generate_series(a), -- a simple scalar expression
   rows from(generate_series(b)), -- a rows from clause that can be elided
   rows from(generate_series(a), generate_series(b)) -- this one not
