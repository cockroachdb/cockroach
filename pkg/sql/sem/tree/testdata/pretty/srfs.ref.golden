// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	*
FROM
	generate_series(
		a
	),
	generate_series(
		b
	),
	ROWS FROM (
		generate_series(
			a
		),
		generate_series(
			b
		)
	)

23:
-----------------------
SELECT
	*
FROM
	generate_series(a),
	generate_series(b),
	ROWS FROM (
		generate_series(
			a
		),
		generate_series(
			b
		)
	)

26:
--------------------------
SELECT
	*
FROM
	generate_series(a),
	generate_series(b),
	ROWS FROM (
		generate_series(
			a
		),
		generate_series(b)
	)

27:
---------------------------
SELECT
	*
FROM
	generate_series(a),
	generate_series(b),
	ROWS FROM (
		generate_series(a),
		generate_series(b)
	)

54:
------------------------------------------------------
SELECT
	*
FROM
	generate_series(a),
	generate_series(b),
	ROWS FROM (generate_series(a), generate_series(b))

94:
----------------------------------------------------------------------------------------------
SELECT
	*
FROM
	generate_series(a), generate_series(b), ROWS FROM (generate_series(a), generate_series(b))

104:
--------------------------------------------------------------------------------------------------------
SELECT * FROM generate_series(a), generate_series(b), ROWS FROM (generate_series(a), generate_series(b))


